src/elastio-snap.c

```diff
@@ -1751,6 +1751,16 @@ static int pathname_concat(const char *pathname1, const char *pathname2, char **
 	return 0;
 }
 
+static inline int is_pathname_exists(const char *pathname){
```

Comment:
Grammatically this is more correct without "is"

```suggestion
static inline int pathname_exists(const char *pathname){
```